tululbot/version.py

```diff
@@ -0,0 +1,5 @@
+VERSION = '1.11.3'
+
+
+def version_formatted():
```

Comment:
Since you can import `VERSION` directly, this function is not really needed isn't it? At this point, the function only preprends `v` to the version number. Also, this variable might be best put under `tululbot/__init__.py` file so that it can be imported like `from tululbot import VERSION`.